base/src/test/java/org/aya/test/LibraryTest.java

```diff
@@ -4,18 +4,31 @@
 
 import org.aya.cli.library.LibraryCompiler;
 import org.aya.util.FileUtil;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
+import java.nio.file.Path;
 
 public class LibraryTest {
   @Test public void test() throws IOException {
-    main();
+    compile();
+    // Run twice, the second time should load the cache directly.
+    compile();
   }
 
```

Comment:
Try run 3 times:
- build all from source
- Remove test out, build test from source, while reusing the common cache
- build all from cache

Reply:
🤔